identity/app/views/consentJourney.scala.html

```diff
@@ -39,51 +39,48 @@ <h3 class="manage-account__switch-title"></h3>
     }
 }
 
-@showAgeStep(index: Option[Int]) = @{
-    if(index.contains(0)){
-        List(ConsentStepHelpLegalText("By clicking the button, you confirm that you are older than 13 years or that you have the consent of your parent or a person holding parental responsibility."))
+@showAgeStep(display: Boolean = false) = @{
+    if(display){
+        List(ConsentStepHelpLegalText("By continuing, you confirm that you are older than 13 years or that you have the consent of your parent or a person holding parental responsibility."))
     } else {
         Nil
     }
 }
 
-@renderStep(step: ConsentStep, index: Option[Int] = None) = {
-    @if(step.show) {
-        @(step.banner.map { title =>
-            Html(s"<p class='form__success'>$title</p>")
-        })
-        <fieldset
-            class="identity-wizard__step"
-            data-wizard-step-name="@step.name"
-            role="dialog"
-            aria-labelledby="consentWizard@{step.name.capitalize}Title"
-        >
-            <legend id="consentWizard@{step.name.capitalize}Title" class="identity-title identity-title--page-title">@step.title</legend>
-            <div class="identity-forms-wrapper">
-                @if(step.help || showAgeStep(index)) {
-                    <div class="identity-forms-wrapper__title">
-                        @((step.help ++ showAgeStep(index)).map(paragraph =>
-                            paragraph match {
-                                case m: ConsentStepHelpLegalText => {
-                                    Html(s"<p class='identity-title-explainer identity-title-explainer--small'>${m.text}</p>")
-                                }
-                                case m: ConsentStepHelpText => {
-                                    Html(s"<p class='identity-title-explainer'>${m.text}</p>")
-                                }
+@renderStep(step: ConsentStep, isFirst: Boolean = false, isLast: Boolean = false) = {
+    @(step.banner.map { title =>
+        Html(s"<p class='form__success'>$title</p>")
+    })
+    <fieldset
+        role="dialog"
+        aria-labelledby="consentWizard@{step.name.capitalize}Title"
+    >
+        <legend id="consentWizard@{step.name.capitalize}Title" class="identity-title identity-title--page-title">@step.title</legend>
+        <div class="identity-forms-wrapper">
+            @if(step.help || showAgeStep(isFirst)) {
```

Comment:
I don'y really understand this branch, what does `showAgeStep(isFirst)` mean?

Reply:
it's the condition to show the age step or not, i'll clarify that!